deepdish/io/hdf5io.py

```diff
@@ -270,7 +280,17 @@ def _load_pickled(level):
 
 def _load_level(handler, level):
     if isinstance(level, tables.Group):
-        dct = {}
+        if _sns and (level._v_title.startswith('sns:') or
+                     DEEPDISH_IO_ROOT_IS_SNS in level._v_attrs):
+            val = sns()
+            dct = val.__dict__
+        elif level._v_title.startswith('list:'):
+            dct = {}
+            val = []
```

Comment:
It isn't necessary. It was really added for handling recursion which will be in the next PR. I'll remove it and go back to the original.